src/server/qgswmsserver.cpp

```diff
@@ -2139,6 +2140,37 @@ void QgsWmsServer::configureMapSettings( const QPaintDevice* paintDevice, QgsMap
   }
 
   mapSettings.setExtent( mapExtent );
+
+  /* Define the background color
+   * Transparent or colored
+   */
+  //is format jpeg?
+  QString format = mParameters.value( QStringLiteral( "FORMAT" ) );
+  bool jpeg = format.compare( QLatin1String( "jpg" ), Qt::CaseInsensitive ) == 0
+              || format.compare( QLatin1String( "jpeg" ), Qt::CaseInsensitive ) == 0
+              || format.compare( QLatin1String( "image/jpeg" ), Qt::CaseInsensitive ) == 0;
```

Comment:
It's a copy of the old code...
We can clean it after merging the qgis server refactoring.